tfe_hardware/qcodes_instrument_drivers/Sinko/AD5760.py

```diff
@@ -0,0 +1,170 @@
+        self.connect_board(host_name, path, syspath)
+
+        self.add_parameter('output',
+                           label='Output State',
+                           get_cmd=self.state,
+                           set_cmd=lambda x: self.on() if x else self.off(),
+                           val_mapping={
+                               'off': 0,
+                               'on': 1,
+                           })
+        
+        self.add_parameter('voltage',
+                           label='Voltage',
+                           unit='V',
+                           set_cmd=self._get_set_volt,
+                           get_cmd=self._get_volt,
+                           vals= Numbers(min_value=-10, max_value=10)
+                           )
+        
+        self.add_parameter('current',
+                           label='Current',
+                           unit='A',
+                           set_cmd=self._get_set_current,
+                           get_cmd=self._get_current,
+                           vals= Numbers(min_value=-0.1, max_value=0.1))
+        
+    def on(self) -> None:
+        self.write('OUTPUT 1')
+        self.measure._output = True
+        
+    def off(self) -> None:
+        self.write('OUTPUT 0')
+        self.measure._output = False
+
+    def state(self):
+        state = self.ask()
+        if state is not None:
+            return bool(1)
+        else:
+            return bool(0)
+    
+    def connect_board(self, host_name: str, path: str, syspath: str):
+        
+        sys.path.append(rf'{syspath}')
+
+        clr.AddReference('AnalogDevices.Csa.Remoting.Clients')
+        clr.AddReference('AnalogDevices.Csa.Remoting.Contracts')
+        
+        # noinspection PyUnresolvedReferences,SpellCheckingInspection
+        from AnalogDevices.Csa.Remoting.Clients import ClientManager  #noqa
+        manager = ClientManager.Create(-1)
+        self.client = manager.CreateRequestClient(f"localhost:{host_name}")
+        self.client.ContextPath = path
+        self.client.WriteRegister("2", "786")
+        
+    def _get_set_volt(self,
+                        output_level: Optional[float] = None
+                        ) -> Optional[float]:
+        """
+        Get or set the output level.
+
+        Args:
+            mode: "CURR" or "VOLT"
+            output_level: If missing, we assume that we are getting the
+                current level. Else we are setting it
+        """
+        if output_level is not None:
+            self.write(output_level)
+            return None
+        return self.ask()
+    
+    def _get_set_current(self,
+                      output_level: Optional[float] = None
+                      ) -> Optional[float]:
+        
+        if output_level is not None:
+            self.write(output_level*100)
+            return None
+        return self.ask()/100
+    
+    
+    def ramp_current(self, ramp_to: float, step: float, delay: float) -> None:
+
```

Comment:
copy the docstring like the voltage one, specifically adding units to delay is helpful since you never know with computer what is the time measurement.